qcengine/programs/model.py

```diff
@@ -76,19 +76,27 @@ def build_input_model(
         # Note: Someday when the multiple QCSchema versions QCEngine supports are all within the
         #  Pydantic v2 API base class, this can use discriminated unions instead of logic.
 
-        if isinstance(data, qcelemental.models.v1.AtomicInput):
-            mdl = model_wrapper(data, qcelemental.models.v1.AtomicInput)
-        elif isinstance(data, qcelemental.models.v2.AtomicInput):
-            mdl = model_wrapper(data, qcelemental.models.v2.AtomicInput)
+        v1_model = getattr(qcelemental.models.v1, "AtomicInput")
```

Comment:
Is there a reason to access `AtomicInput` this way rather than though an import? If so, it would be helpful to add a comment to explain why that is necessary.

Reply:
Not a particularly good reason -- I switched in here in programs/model.py (where it's always `AtomicInput`) so the analog was clearer with procedures/model.py https://github.com/MolSSI/QCEngine/pull/458/files#diff-2e40dbc50862d2bd8716a45e9dafab44cb3be4499b972dc1d58b8144ef1ab046R63 where the flavor or Input isn't fixed.